### DISCUSSION TITLE
A question from a front-end beginner

### DISCUSSION THREAD

**Discussion by KelseyOnGit** [community/community#77140] [Programming Help] [answered]
### Body

Please help me with this 
I'm a total beginner learning basic language(html,css,javascript)
I try to link my css to html so as to style my page, but it doesn't work.
And I cannot figure out what's wrong with my coding.
I will really appreciate if you can help me out!

![2023-11-28_8-59-37](https://github.com/community/community/assets/126480141/55f32ee5-dcdc-4617-b4fe-42610807a4a8)
![2023-11-28_8-59-51](https://github.com/community/community/assets/126480141/b1d80535-41fa-406d-a543-0e3d6589bd4b)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by KelseyOnGit** [comment]
Add: It's not a link issue. I clicked the link in my html page and the css page would pop up

**Reply by ihnatenkoo** [reply, marked as answer]
Hello
Copy this to your index.html

```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
		<title>Document</title>
		<link rel="stylesheet" href="styles.css" />
	</head>
	<body>
		<p>I am teaching English</p>
	</body>
</html>
```


then create styles.css file near your index.html and copy this:
`p { color: aquamarine; } `

**Reply by KelseyOnGit** [reply]
It works! Thanks a lot for your help.

**Comment by mdjumman899** [comment]
Aviator

**Comment by mdjumman899** [comment]
<!DOCTYPE html>
<html lang="bn">
<head>
  <meta charset="UTF-8">
  <title>Aviator গেম</title>
  <style>
    body { font-family: Arial; background-color: #0f0f0f; color: white; text-align: center; padding: 50px; }
    h1 { color: red; }
    a.button { background: #ff3333; padding: 10px 20px; color: white; text-decoration: none; border-radius: 5px; }
  </style>
</head>
<body>
  <h1>অ্যাভিয়েটর গেমে স্বাগতম!</h1>
  <p>এই গেমে আপনি multiplier বাড়ার আগেই ক্যাশ আউট করতে পারবেন।</p>
  <a class="button" href="#">গেম শুরু করুন</a>
</body>
</html>
